packages/expo-modules-core/ios/JSI/EXJavaScriptRuntime.mm

```diff
@@ -105,7 +105,7 @@ - (nonnull EXJavaScriptObject *)global
 
 - (nonnull EXJavaScriptValue *)evaluateScript:(nonnull NSString *)scriptSource
 {
-  std::shared_ptr<jsi::StringBuffer> scriptBuffer = std::make_shared<jsi::StringBuffer>([[NSString stringWithFormat:@"(%@)", scriptSource] UTF8String]);
+  std::shared_ptr<jsi::StringBuffer> scriptBuffer = std::make_shared<jsi::StringBuffer>([scriptSource UTF8String]);
```

Comment:
I was wrong with wrapping it in parentheses. I assumed it would be helpful since objects and functions are not returned by `eval` when they are not wrapped, however that may break other code, especially if it contains `;`